UI/obs-app.cpp

```diff
@@ -228,7 +228,7 @@ string CurrentTimeString()
 
 	auto tp = system_clock::now();
 	auto now = system_clock::to_time_t(tp);
-	tstruct = *localtime(&now);
+	tstruct = *localtime_r(&now);
```

Comment:
FWIW, the whatever_r() functions are never drop-in replacements for their non-reentrant siblings